### ISSUE TITLE
Disable Codacy in CI? 

### ISSUE THREAD

**Issue by traversaro** [robotology/gazebo-yarp-plugins#561]
In my experience, the [signal-to-noise ratio](https://en.wikipedia.org/wiki/Signal-to-noise_ratio) of Codacy is not big enough that it make sense to keep it enabled. There is anyone that strongly feels that instead we should keep it enabled?

**Comment by lrapetti** [comment]
I agree on disabling it. Back in time, we removed it from [`human-dynamics-estimation`](https://github.com/robotology/human-dynamics-estimation) repo for the same reason (e.g. see https://github.com/robotology/human-dynamics-estimation/pull/164#issuecomment-579253337 and  https://github.com/robotology/human-dynamics-estimation/search?q=codacy&type=issues)
